enterprise/utils.py

```diff
@@ -1792,17 +1796,31 @@ def customer_admin_enroll_user_with_status(
     succeeded = False
     new_enrollment = False
     enterprise_fulfillment_source_uuid = None
+    emet_enable_auto_upgrade_enrollment_mode = get_configuration_value(
+        'FEATURE_ENABLE_EMET_AUTO_UPGRADE_ENROLLMENT_MODE',
```

Comment:
[clarification] Where does `get_configuration_value` pull its config settings from? I believe `FEATURE_ENABLE_EMET_AUTO_UPGRADE_ENROLLMENT_MODE` is only defined for the MFE configuration API under the `MFE_CONFIG_OVERRIDES` setting, which afaik is different than where `get_configuration_value` pulls from.

Out of curiosity, I'm wondering whether we _need_ to introduce a conditional feature flag for supporting audit -> verified enrollment upgrades? What is the harm/risk in not including a feature flag on the API side of things? 

The only reason the audit -> verified upgrade code path has a feature flag with `FEATURE_ENABLE_EMET_AUTO_UPGRADE_ENROLLMENT_MODE` is because the API didn't support it yet. But, if the API supports it (without a feature flag), in theory I believe we could then remove `FEATURE_ENABLE_EMET_AUTO_UPGRADE_ENROLLMENT_MODE` altogether on the frontend side of things. 

For example, we might implement these API changes without a feature flag, enable the feature flags on stage then production, and then once verified, remove `FEATURE_ENABLE_EMET_AUTO_UPGRADE_ENROLLMENT_MODE` altogether from frontend-app-learner-portal-enterprise.

Reply:
I think you are correct that `get_configuration_value` won't have access to `FEATURE_ENABLE_EMET_AUTO_UPGRADE_ENROLLMENT_MODE`, so will revert that change!
I remember that there will be some time between when the changes in `edx-enterprise` are available in the new release and when the changes are available in staging or prod in `edx-platform`. I wanted to be able to fall back onto the old `lms_enroll_user_in_course` function while there is a bit of time misalignment between the two deployments. Perhaps a feature flag on the API side would work best?

Reply:
Following up from Slack conversation about this feature flag, I didn't realize the coupling between edx-platform and edx-enterprise at the time. I see why introducing a temporary feature flag here makes sense :) Feel free to ignore this comment thread now.